Makefile

```diff
@@ -41,20 +41,17 @@ build:
 	DOCKER_BUILDKIT=1 docker build --tag $(IMAGE) --target $(DOCKER_IMAGE_TAG)  --build-arg record=$(record) .
 
 build_venv:
-	python3 -m venv mava_env
-	mava_env/bin/pip install git+https://github.com/RuanJohn/jumanji.git@vmap-ippo
-	mava_env/bin/pip install -r https://raw.githubusercontent.com/luchris429/purejaxrl/main/requirements.txt
-	mava_env/bin/pip install jax==0.4.9
-	mava_env/bin/pip install jaxlib==0.4.7+cuda11.cudnn82  -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
-	echo 'export PYTHONPATH="./"' >> mava_env/bin/activate
+	python3 -m venv mava_30
```

Comment:
shouldn't it be mava_3-0?

Reply:
Yes I agree it looks weird. Maybe we should just call it `mava_env`? It might be easier. What do you think?

Reply:
Updated :)